packages/terra-clinical-item-collection/README.md

```diff
@@ -1,5 +1,11 @@
 # Terra Clinical Item Collection
 
+## Deprecation Note
+
+This package is no longer supported as it was too limited to meet the varied needs of all consumers. Consumers are instead recommonded to create their own flexible list/table component as per their needs instead.
```

Comment:
Should we link to the guide as is done here? https://engineering.cerner.com/terra-ui/components/terra-ui/clinical-item-collection/clinical-item-collection

Reply:
I don't know how long we'll have that page so I don't want people to end up on a 404 page.

Reply:
wait - why would that page go away?

Reply:
Will we have keep that page forever? I just thought we might delete that page some time after we delete the component.

Reply:
Won't consumers still potentially want/need to reference the "Building an Item Collection" guide, though?

Reply:
Hmmm, looks like we still have weekly downloads so yeah, it would be prudent to link to the guide directly. I'll add that. Good call.

Reply:
> I don't know how long we'll have that page so I don't want people to end up on a 404 page.

I just realized that I mixed up the pages so we can disregard this since we were talking about linking the guide directly.

Reply:
Updated here: 9e3135045a3edfcd729e8a366d11aa231ed5c3af